### PR TITLE
feat: add subgraph with example

### PR DESCRIPTION
**Pull request by 2manslkh**

### CODE REVIEW

**Comment by dionysuzx** [comment]
i'm seeing this on preview @2manslkh 

image

**Review by dionysuzx — APPROVED** [review]

File: packages/app/src/utils/graph.ts [hunk 1]
```diff
@@ -0,0 +1,8 @@
+import { Client, cacheExchange, fetchExchange } from '@urql/svelte';
+
+const APIURL = ' http://43.153.69.251:8000/subgraphs/name/taiko/diamondsNFT'
```
Comment (dionysuzx) [inline packages/app/src/utils/graph.ts]:
```suggestion
const API_URL = 'http://43.153.69.251:8000/subgraphs/name/taiko/diamondsNFT'
```

File: packages/app/src/utils/graph.ts [hunk 2]
```diff
@@ -0,0 +1,8 @@
...
+
+export const client = new Client({
+    url: APIURL,
```
Comment (dionysuzx) [inline packages/app/src/utils/graph.ts:7]:
```suggestion
    url: API_URL,
```

File: packages/app/wagmi.config.ts [hunk 3]
```diff
@@ -1,15 +1,41 @@
...
+  // loop through out and create a ContractConfig for each contract
+  // let contracts: ContractConfig<number, undefined>[] = []
+  // Object.keys(out).forEach((contractName) => {
+  //   contracts.push({
+  //     name: contractName,
+  //     address: out[contractName].address,
+  //     abi: out[contractName].abi,
+  //   });
+  // });
+  // return contracts;
```
Comment (dionysuzx) [inline packages/app/wagmi.config.ts:36]: should we keep this comment here

**Comment by 2manslkh** [comment]
Currently the subgraph API URL has to be https before vercel can query it

**Pull request merged**